lib/versioning/poetry/index.spec.ts

```diff
@@ -119,9 +120,11 @@ describe(getName(), () => {
       expect(versionig.isSingleVersion('1.2.3')).toBeTruthy();
       expect(versionig.isSingleVersion('1.2.3-alpha.1')).toBeTruthy();
     });
-    it('returns true if equals', () => {
-      expect(versionig.isSingleVersion('=1.2.3')).toBeTruthy();
-      expect(versionig.isSingleVersion('= 1.2.3')).toBeTruthy();
+    it('supports only PEP440 exact versions', () => {
```

Comment:
Depends.. does poetry support this somehow? Maybe in this case it's a "constraint" and not a version?

Reply:
![image](https://user-images.githubusercontent.com/1798109/117663092-8dfd3f00-b1a0-11eb-948c-6b4b3019bd13.png)
https://python-poetry.org/docs/versions/#exact-requirements

We should follow those docs